R/import.R

```diff
@@ -55,8 +64,58 @@ mintr_db_process <- function(path) {
   prevalence$netType <- relevel(prevalence$netType, c(std = 1, pto = 2))
   prevalence$intervention <- relevel(prevalence$intervention,
                                      import_intervention_map())
-
+  prevalence$year <- NULL
   saveRDS(prevalence, paths$prevalence)
+
+  message("Processing table")
+  path_table_raw <- file.path(path, raw$directory, raw$files$table)
+  table <- readRDS(path_table_raw)
+
+  tr <- c(netUse = "switch_nets",
+          irsUse = "switch_irs",
+          netType = "NET_type",
+          casesAverted = "cases_averted",
+          prevYear1 = "prev_1_yr_post",
+          prevYear2 = "prev_2_yr_post",
+          prevYear3 = "prev_3_yr_post",
+          reductionInPrevalence = "relative_reduction_in_prevalence",
+          reductionInCases = "relative_reduction_in_cases",
+          meanCases = "cases_per_person_3_years")
+  table <- rename(table, unname(tr), names(tr))
+
+  ## At this point casesAverted is really over 3 years and is cases
+  ## averted per person. This number can be greater than one as a
+  ## person can have more than one case per year. We remove the year
+  ## effect first:
+  table$casesAverted <- table$casesAverted / 3
+
+  ## Then compute the "per 1000" case before the uncertainty calculation:
+  table$casesAvertedPer1000 <- table$casesAverted * 1000
+
```

Comment:
Can we round both of the above to the nearest whole number? And also have:

```
  # We want these as percentages, rounded to the nearest d.p.
  table$reductionInCases <- round(table$reductionInCases * 100, 1)
  table$reductionInPrevalence <- round(table$reductionInPrevalence * 100, 1)
```

Reply:
added this now

Reply:
Brill, thanks. Can we also round casesAverted and casesAvertedPer1000 to the nearest whole number?